atlassian_jwt_auth/contrib/server/__init__.py

```diff
@@ -0,0 +1,45 @@
+import logging
+
+import jwt.exceptions
+import requests.exceptions
+
+from atlassian_jwt_auth.exceptions import (
+    PublicKeyRetrieverException,
+)
+
+
+def _requires_asap(verifier, auth, parse_jwt_func, response_class,
```

Comment:
Nit: `__init__.py` probably shouldn't have code that isn't related to importing.

Maybe just have `server.py` or `server/helpers.py` instead?

Reply:
Sure.